docs/rfc/ethereum_attestation.md

```diff
@@ -0,0 +1,126 @@
+# RFC: Ethereum attestation
+
+* Author: @CodeSandwich
+* Date: 2021-03-05
+* Status: draft
+* Community discussion: n/a
+
+## Motivation
+
+The attestation between Link and Ethereum is a valuable building block for a user identity.
+It brings the Link reputation coming from projects and contributions to the
+Ethereum world of DAOs and donations, where it's important to know, who's behind an address.
+On the other hand it lends Ethereum account reputation with its assets and undeniable history
+to Link to build user's trust in the identity.
+
+## Overview
+
+This RFC is built on top of [Identity Proofs RFC][rfc].
+It introduces support for Ethereum address claims on Link
+and a smart contract on Ethereum to make Link identity claims on Ethereum.
+
+## Link identity JSON extension
+
+The identity JSON supports a new key: `ethereum`.
+Under this key there is stored an ethereum address claim following this convention:
+
+- `account` - the claimed ethereum address, encoded according to [EIP-55][eip-55],
+e.g. using [ethers.js][ethers-addr]
+- `proof` - always `null`
+
+Example:
+```json
+{
+    "ethereum": {
+        "account": "0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B",
+        "expiration": {
+            "created": 1614768373,
+            "expires": 31536000
+        },
+        "proof": null
+    }
+}
+```
+
+## Ethereum smart contract
+
+A new smart contract is deployed to the network, which is used to publish link claims:
+
+```solidity
+contract Claims {
+    event Claimed(address indexed addr);
+    function claim(uint8 version, uint256 id) public;
+}
+```
+
+In order to claim an ID, call `claim` using your Ethereum account.
+It will emit an event `Claimed`, which later can be queried to discover your attestation.
+The claims have no expiration date and don't need to be renewed.
```

Comment:
This means that the link can be revoked with only the ethereum account, right? Is that desirable? Basically, since the Link identity doesn't point to a specific attestation, it can be revoked from the ethereum side, without having the Link key.

Reply:
That's right, if you loose your Link key, you need to be able to revoke the attestation using only your Ethereum key.

Reply:
The payload above has an expiration date. So this "claim" seems to be false :smirk: